Missing #include<string> MSVC

When trying to build the project using the MSVC compiler version 19.41.34120.0 the build fails with a whole bunch of errors of the type
`faker-cxx\include\faker-cxx\airline.h(89,23): error C2039: 'string': is not a member of 'std'
 [C:\Users\\Documents\faker-cxx\build\faker-cxx.vcxproj]
  (compiling source file '../src/modules/airline.cpp')
      C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.41.34120\include\__msvc_string_view.hpp
  (22,1):
      see declaration of 'std'`
This happens for all files that only include `string_view` but use `std::string`. Including `string` solves the problem and the library builds.

--- a/include/faker-cxx/airline.h
+++ b/include/faker-cxx/airline.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/commerce.h
+++ b/include/faker-cxx/commerce.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/company.h
+++ b/include/faker-cxx/company.h
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <optional>
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/database.h
+++ b/include/faker-cxx/database.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/date.h
+++ b/include/faker-cxx/date.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/hacker.h
+++ b/include/faker-cxx/hacker.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/image.h
+++ b/include/faker-cxx/image.h
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <optional>
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/location.h
+++ b/include/faker-cxx/location.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/person.h
+++ b/include/faker-cxx/person.h
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <optional>
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

--- a/include/faker-cxx/vehicle.h
+++ b/include/faker-cxx/vehicle.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
 #include <string_view>
 
 #include "faker-cxx/export.h"

